src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -44,6 +52,11 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
         const { expression, returnStatement, func } = info;
 
         let body: ConciseBody;
+
+        // These two variables are used if there is a trailing comment after the returnStatement
+        let newEdit: TextChange;
+        const trailingCommentsHolder = createStringLiteral("");

```

Comment:
I wasn't sure the best way to create an "empty token" essentially. I needed this because (as you'll see below) I'm using a utility function someone else wrote that copies a trailing comment from one token to another token. So this `trailingCommentsHolder` is meant to be that token where I copy the comment to.